lib/ansible/executor/task_executor.py

```diff
@@ -134,7 +134,10 @@ def _clean_res(res):
                 if isinstance(res, UnsafeProxy):
                     return res._obj
                 elif isinstance(res, binary_type):
-                    return to_text(res, errors='surrogate_or_strict')
+                    try:
+                        return to_text(res, errors='surrogate_or_strict')
+                    except:
+                        return bytes(res)
```

Comment:
@abadger maybe I can try to remove bytes with there respective hex representation like this:

```Un super fichier avec \xe9 en iso8859-15

A super file with \xe9 in iso8859-15```

What do you think about?